File: Modul 4/Bahasa Python (modul 4)/PRAK403-2310817320015-AlfisyahRinaFajriati.py
print("Input")
bil1, bil2 = input().split()
a = int(bil1) ; b = int(bil2)

print("\nOutput")
if a > b :
    while a >= int(bil2) and b <= int(bil1):
        print(a, end=" ")
        print(b, end=" ")
        a -= 1 
        b += 1
        if a == int(bil2) - 1: 
            break
        else : 
            print("-", end=" ")

elif a < b :
    while a <= int(bil2) and b >= int(bil1):
        print(a, end=" ")
        print(b, end=" ")
        a += 1
        b -= 1
        if b == int(bil1) - 1: 
            break
        else: 
            print("-", end=" ")

File: Modul 4/Bahasa C (modul 4)/PRAK403_2310817320015_AlfisyahRinaFajriati.c
#include <stdio.h>
int main() {
    printf("Input\n");
    int bil1, bil2, a, b;
    scanf("%d %d", &bil1, &bil2);

    printf("\nOutput\n");
    if (bil1 > bil2){
        for (a = bil1, b = bil2; a >= bil2, b <= bil1; a--, b++){
            printf ("%d %d ", a, b);
            if (a == bil2){
               break ; }
            else {
                printf("- "); }}}
    else {
        for (a = bil1, b = bil2; a <= bil2, b >= bil1; a++, b--){
            printf("%d %d ", a, b);
            if (a == bil2){
               break ; }
            else {
                printf("- "); }}}

    return 0;
}